### ISSUE TITLE
Some oracle responses are causeing server error

### ISSUE THREAD

**Issue by janmichek** [aeternity/ae_mdw#1307]
taking some oracles from listing 
https://mainnet.aeternity.io/mdw/v2/oracles

This one works as expected 
https://mainnet.aeternity.io/mdw/v2/oracles/ok_qJZPXvWPC7G9kFVEqNjj9NAmwMsQcpRu6E3SSCvCQuwfqpMtN/responses

but this one is throwing error
https://mainnet.aeternity.io/mdw/v2/oracles/ok_2fVpBb8j58mNQcVjY7bwUVpBWZv6AySGDjqepNofU8XGGmEN53/responses

**Comment by janmichek** [comment]
Nope, sorry was mixing the ids from testnet and mainnet. All seems to be good now

**Comment by janmichek** [comment]
Sorry again, this is still valid.

**Comment by sborrazas** [comment]
Fixed
